css/style.css

```diff
@@ -641,20 +652,468 @@ hr {
+  margin-left: 0;
+}
+
+@media (min-width: 768px) {
+  body {
+    width: wider;
+    height: 72px;
+  }
+
+  nav {
+    color: #fff;
+    margin: 2% 2%;
+  }
+
+  /* .header {
+    width: 1440px;
+    height: 72px;
+  } */
+
+  .fa {
+    display: none;
+  }
+
+  .hidden {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 50px;
+    right: 100px;
+    top: calc(6% - 40px / 2);
+  }
+
+  .hidden div {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  .hidden a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .content {
+    display: grid;
+    width: 100%;
+    height: 870px;
+    background-image: url("/images/header-img-2.png");
+    background-repeat: no-repeat;
+    background-position: 100% 100%;
+    z-index: 1;
+  }
+
+  .wrapper > h1 {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    width: 566px;
+    height: 160px;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .wrapper > h4 {
+    display: flex;
+    width: 566px;
+    height: 24px;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    align-items: center;
+  }
+
+  .wrapper > p {
+    display: flex;
+    width: 566px;
+    height: 123px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    align-items: center;
+    flex: none;
+  }
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    position: absolute;
+    width: 566px;
+    height: 100%;
+    left: 143px;
+  }
+
+  .wrapper-2 {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    align-items: center;
+    margin-top: 89px;
+    width: 18px;
+  }
+
+  .icons {
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    position: absolute;
+  }
+
+  .recent-work {
+    padding: 130px 118px;
+    width: 100%;
+    height: 1805px;
+  }
+
+  .title {
+    width: 100%;
+    height: 52px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .h-title {
+    align-items: flex-start;
+    height: 50px;
+  }
+
+  #h-line {
+    display: flex;
+    width: 70%;
+    justify-content: center;
+    flex-direction: row;
+    align-items: center;
+    color: #212121;
+    background-color: #212121;
+  }
+
+  .recent-frame-1 {
+    margin-top: 69px;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+  }
+
+  .post-img {
+    width: 684px;
+    height: 448px;
+  }
+
+  .post-details {
+    display: flex;
+    line-height: 0;
+    height: 10px;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .recent-frame-2 {
+    display: grid;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 100%;
+    height: 0;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .card-work {
+    padding: 0;
+    margin: 0;
+  }
+
```

Comment:
- [ ] I really appreciate your work, in fact, you really have done a great job to make your design responsive while sticking to the design you chose and that's great👏. However, there is something you forgot to add that I think is very important for your portfolio. It is about your project cards. They look great but you forgot to add user interaction on them. See initially there should be **an overlay effect with texts describing the project** on each of your project cards(I mean those six aligned with `grid`), then when the user hovers on any of them, **the overlay effects and texts are supposed to disappear** and the user sees only the project image and the `see project` button.
**Your Design**
![image](https://user-images.githubusercontent.com/90222110/177874333-ac8d0cf0-693e-44e2-91d7-5e37657e63c7.png)
Looking at your design, you can see what the **first project card** looks like. **Each of your project cards is supposed to look like that when the user hovers on it** otherwise, there **should be that overlay effect with the project description texts as it is on other project cards** 🚀. These are important details that a developer should take into account because it is what makes him/her outstanding 😃. Please implement this feature so as to make your portfolio more professional. Let me hope that my explanations are really understandable 😺 .